### PR TITLE
fix: configuration_installed

### PR DESCRIPTION
**Pull request by zhangsoledad**
* fix configuration_installed 
* fix capacity estimates in readme, now `create_udt_account_wallet_cell` need 3010 capacity.
* remove `UdtWallet#calculate_cell_min_capacity`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ Now we can perform normal transfers between wallets:
 [7] pry(main)> alice.get_balance
 => 12345
 [8] pry(main)> bob.get_balance
-=> 337655
+=> 87655
```
Comment (ashchan) [inline README.md]: 👍

File: lib/ckb/always_success_wallet.rb [hunk 2]
```diff
@@ -16,14 +16,14 @@ def send_capacity(target_address, capacity)
       outputs = [
         {
           capacity: capacity,
-          data: [],
+          data: "",
```
Comment (ashchan) [inline lib/ckb/always_success_wallet.rb]: What made us change data from `[uint8]` to `String`?

Comment (zhangsoledad) [reply lib/ckb/always_success_wallet.rb]: Data be packed to binary string, such as `[i.amounts - amount].pack("Q<")`,
also see `Ckb::Utils.calculate_cell_min_capacity`  `output[:data].bytesize`

Comment (xxuejie) [reply lib/ckb/always_success_wallet.rb]: I think right now CKB's RPC still only accepts array of integers as `Vec<u8>` correct? Or have we changed that?

Comment (zhangsoledad) [reply lib/ckb/always_success_wallet.rb]: Ok, I am confused with two format, so we use binary string for sha, array byte for RPC request ?
I will revert this, leave it for the future.

**Pull request merged**